src/app/app.module.ts

```diff
@@ -34,13 +42,19 @@ import { environment } from '../environments/environment';
     CampaignModule,
     BrowserAnimationsModule,
     DashboardModule,
+    MatMomentDateModule,
     StoreModule.forRoot(reducers, { metaReducers }),
     StoreModule.forRoot({ router: routerReducer }),
     StoreRouterConnectingModule.forRoot({ serializer: CustomRouterSerializer }),
     EffectsModule.forRoot([]),
     !environment.production ? StoreDevtoolsModule.instrument() : []
   ],
-  providers: [{ provide: ErrorHandler, useClass: ErrorService }],
+  providers: [
```

Comment:
I guess DataAdapters don't work unless applied in the app.module. I had this in the campaign.module previously, and that's why it wasn't working. =/